test/templates/dialog/index.njk

```diff
@@ -0,0 +1,209 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+{% from "./component.njk" import dialog %}
+{% from "../icon/component.njk" import icon %}
+{% set css_deps = ['fonts','components/button','icons','layout/overlay'] %}
+
+{% block content %}
+
+<h1>dialog</h1>
+
+
+{% set exampleSmall %}
+
+
+  {{  dialog(
+        properties={
+            header: "Modal dialog",
+            class: "fd-dialog__header",
+            size: "small",
+            body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
+            actions: [
+              {type: 'emphasized', label: 'Confirm'},
+              {type: 'secondary', label: 'Cancel'}
+            ]
+          }
+      )
+  }}
+
+{% endset %}
+
+{% set errorExampleSmall %}
+
+{{ dialog(
+        properties={
+          icon: "sap-icon--message-error",
+          header: "Error Message",
+          class: "fd-dialog__header-error",
+          size: "small",
+          body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
+          actions: [
+            {type: 'emphasized', label: 'Confirm'},
+            {type: 'secondary', label: 'Cancel'}
+          ]
+        }
+    ) 
+}}
+{% endset %}
+
+{% set warningExampleSmall %}
+
+{{ dialog(
+        properties={
+          icon: "sap-icon--message-warning",
+          header: "Warning Message",
+          class: "fd-dialog__header-warning",
+          size: "small",
+          body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
+          actions: [
+            {type: 'emphasized', label: 'Confirm'},
+            {type: 'secondary', label: 'Cancel'}
+          ]
+        }
+    ) 
+}}
```

Comment:
Preferred setup is ...

```
  {{  dialog(
         properties={
             header: "Modal dialog",
             body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentes vitae erat feugiat, cursus lorem a condimentum sem.",
             actions: [
               {type: 'emphasized', label: 'Confirm'},
               {type: 'secondary', label: 'Cancel'}
             ]
           },
           modifier={
           	  block: "small",
           	  header: "warning"
           }
       )
   }}
```

This keep presentation related data apart from the content.